src/Home.jsx

```diff
@@ -13,7 +13,19 @@ import { getActors, getScriptTitles } from './components/api';
 // Load the actor names and script titles on page load
 export async function loader() {
   const [loadedActors, loadedScripts] = await Promise.all([getActors(), getScriptTitles()])
-  return { loadedActors, loadedScripts}
+  if (loadedActors.success && loadedScripts.success){
+    return { loadedActors: loadedActors.data, loadedScripts: loadedScripts.data }
+  }
+  else {
+    if (loadedActors.success === false) {
+      console.error(loadedActors.data)
+      return {loadedScripts: loadedScripts.data}
+    }
+    else {
+      console.error(loadedScripts.data)
+      return {loadedActors: loadedActors.data}
+    }
+  }
```

Comment:
Is it worth it to partially load the data vs not load at all? What will the user experience be in this case?

Reply:
Good point, either way there will be an error since components further down will try to use that data and error out.